creator-node/src/utils/getRegisteredDiscoveryNodes.ts

```diff
@@ -0,0 +1,85 @@
+import type Logger from 'bunyan'
+import { Redis } from 'ioredis'
+const asyncRetry = require('./asyncRetry')
+
+const THIRTY_MINUTES_IN_SECONDS = 60 * 30
+
+type Service = {
+  owner: string
+  endpoint: string
+  spID: number
+  type: 'discovery-node' | 'content-node'
```

Comment:
nit - maybe we restrict to only `discovery-node` for now

Reply:
for sure thx